common/djangoapps/util/views.py

```diff
@@ -84,6 +84,7 @@ def inner(request, *args, **kwargs):
                     # In debug mode let django process the 500 errors and display debug info for the developer
                     raise
                 else:
+                    log.exception("Error in django view.")
```

Comment:
I know this is a generic handler, but since we have the opportunity it seems like we might want to include a more descriptive exception message -- is there any other context we could provide (aside from the stack trace) that would be helpful?